No more entities available after HA Core Update 2024.1.0

Iminet72 opened the pull request:
https://github.com/PlusPlus-ua/ha_mobilealerts/issues/23#issuecomment-1882555904
Thanks for the fix [gainb80](https://github.com/gianb80)

Iminet72 committed:
Update base.py

@dataclasses.dataclass(frozen=True)

Iminet72 committed:
Update sensor.py

self

Iminet72 committed:
Update sensor.py

Iminet72 committed:
Update binary_sensor.py

Teszt dataclass

Iminet72 committed:
Update binary_sensor.py

ky change

Iminet72 committed:
Update sensor.py

key

Iminet72 committed:
Update binary_sensor.py

key

Iminet72 committed:
Update sensor.py

Iminet72 committed:
Update binary_sensor.py

key2

Iminet72 committed:
Update sensor.py

key

Iminet72 committed:
Update sensor.py

Iminet72 committed:
Update sensor.py

Iminet72 committed:
Update sensor.py

state_class=SensorStateClass.MEASUREMENT

Iminet72 committed:
Update sensor.py

lost ","

Iminet72 committed:
Update sensor.py

forgot a comma

Iminet72 committed:
Update sensor.py

..... forgot a comma

Iminet72 committed:
Update sensor.py

Rain sensor duplacate class delete

marek-veber commented:
I have solved probably the same problem :D
 https://github.com/PlusPlus-ua/ha_mobilealerts/pull/25

berndh11 commented:
Thank you for the great fix! Now everything is working again. THANK YOU!

Iminet72 committed:
Update __init__.py

Multi Gateways

unbtx commented:
I had to ommit the change in init.py. Otherwise I get:
2024-11-21 18:13:52.308 ERROR (MainThread) [mobilealerts.gateway] Error resending request to cloud: ClientProxyConnectionError(ConnectionKey(host='192.168.178.75', port=36337, is_ssl=False, ssl=True, proxy=None, proxy_auth=None, proxy_headers_hash=None), ConnectionRefusedError(111, "Connect call failed ('192.168.178.75', 36337)"))

Now both is working: I have the sensor values in HA and data is still send to the cloud of mobile alerts / data199.com

Sergeant-Steve committed:
Update sensor.py

Update to fix "dictionary size changed during iteration" error in Home Assistant.

Sergeant-Steve committed:
Update binary_sensor.py

Update "IsRaining" sensor to work properly in Home Assistant.

Sergeant-Steve committed:
Update const.py

Fix for deprecated enum backport in Home Assistant >2024.6 which now uses Python 3.12.

Sergeant-Steve committed:
Update sensor.py

Update to fix "dictionary size changed during iteration" error in Home Assistant.

Iminet72 committed:
Merge pull request #1 from Sergeant-Steve/main

Update deprecated code, dictionary size issue & fix rain sensor issue

Sergeant-Steve committed:
Update binary_sensor.py

Fix issue with "IsRaining" sensor so it actually updates when it is Wet or Dry and doesn't throw an error.

Sergeant-Steve committed:
Update const.py

Update deprecated code.

Sergeant-Steve committed:
Merge branch 'main' into patch-1

Sergeant-Steve committed:
Merge pull request #1 from Sergeant-Steve/patch-1

Update binary_sensor to properly fix "IsRaining" sensor.

Iminet72 committed:
Merge pull request #2 from Sergeant-Steve/main

Change to binary_sensor.py to actually fix "IsRaining" sensor
